charts/policy-reporter/values.yaml

```diff
@@ -161,6 +161,23 @@ target:
     skipExistingOnStartup: true
     # Added as additional labels to each Loki event
     customLabels: {}
+    # Filter Results which should send to this target by namespaces, priorities or policies
+    # Wildcars for namespaces and policies are supported, you can either define exclude or include values
+    # Filters are available for all targets except the UI
+    filter: {}
+#      namespaces:
+#        include: ["develop"]
+#      priorities:
+#        exclude: ["debug", "info", "error"]
+    channels: []
+#    - host: "http://loki.loki-stack:3100"
+#      sources: []
+#      customLabels: {}
+#      filter:
+#        namespaces:
+#          include: ["develop"]
+#        priorities:
+#          exclude: ["debug", "info", "error"]
```

Comment:
Would you want to add an example policies filter here too?

Reply:
Thanks for the suggestion, I can also add a policy example at this point.